people/matthiasbeyer.toml

```diff
@@ -1,3 +1,4 @@
 name = "Matthias Beyer"
+email = "mail@matthiasbeyer.de"
```

Comment:
This is not my E-Mail address!

Reply:
I just send out a fixup patch to @spacekookie for this.

Reply:
You made me realise I need a better workflow for applying patches to a project directly in emacs with notmuch :see_no_evil: